### ISSUE TITLE
Update requirements.txt versions so packages are no longer pinned to a specific version

### ISSUE THREAD

**Issue by sasjowood** [sassoftware/viya4-ark#113]
We should move away from pinning requirements to a specific version in requirements.txt. This will allow packages containing a fix to be installed without having to update this project to adjust the accepted version.

**Comment by kevinlinglesas** [comment]
@sasjowood is this just asking for there to be no instances of 'pkgA==x.y' and instead make everything 'pkgA>=x.y'?

**Comment by sasjowood** [comment]
@kevinlinglesas, yes I believe that was the goal. It's been awhile now, but I think the impetus was a security fix for Jinja2. Generating a requirements.txt using pip just pins dependencies to their current version. However, we would have picked up the security fix for Jinja2 if we allowed patch-level updates. Does that ring a bell? I might be making that up :).

**Comment by kevinlinglesas** [comment]
I recall a couple GitHub Dependabot notifications about our requirements.txt versions.   Atm, there seems to just be one package that is pinned to a specific version.   This enhancement may be very simple ;)

**Comment by kevinlinglesas** [comment]
https://github.com/sassoftware/viya4-ark/search?q=dependabot&type=issues

**Comment by kevinlinglesas** [comment]
We are choosing to not implement this enhancement at this time.   If we encounter alerts from GitHub dependabots regarding this (not sure such alerts exist), then we will update the one pinned version (ldap3) against newer versions.    We will also consider it a best practice going forward to try and avoid pinning package to a given version unless absolutely necessary.
